docs/source/conf.py

```diff
@@ -22,7 +22,7 @@
 author = 'Stuart Lynn, Milos Colic'
```

Comment:
add @edurdevic, @robertwhiffin, @tiems90 
we should add authorship to all contributors with solutions and code contribution.